libs/errors/website/src/lib/apollo-error-alert.tsx

```diff
@@ -0,0 +1,55 @@
+import {ApolloError} from '@apollo/client/errors'
+import {ComponentProps, PropsWithChildren, useMemo} from 'react'
+import {ErrorCode} from 'errors'
+import {Alert as MuiAlert, Alert} from '@mui/material'
+import {Link} from '@wepublish/ui'
+
+/**
+ * This function is intended to replace with standard i18n translation functions.
+ * @param errorCode
+ */
+export function translateApolloErrorCode(errorCode?: ErrorCode): string | undefined {
+  if (!errorCode) return
+  switch (errorCode) {
+    case ErrorCode.EmailAlreadyInUse:
+      return 'Es besteht bereits ein Konto mit dieser E-mail.'
+    case ErrorCode.ChallengeFailed:
+      return 'Die Rechenaufgabe des Spam-Schutz wurde nicht korrekt gelöst. Bitte versuch es nochmals.'
+    default:
+      return undefined
+  }
+}
+
+export interface ApolloErrorAlertProps extends PropsWithChildren<ComponentProps<typeof MuiAlert>> {
```

Comment:
Probably should live inside the @wepublish/website/builder library like the other interfaces.